src/zenml/orchestrators/step_launcher.py

```diff
@@ -313,14 +310,15 @@ def _run_step(
                     step_run_info=step_run_info,
                 )
             else:
-                self._run_step_without_step_operator(
+                runner = StepRunner(step=self._step, stack=self._stack)
```

Comment:
I would still keep this in a separate method, it's a nice separation and prevents messy code in the future by adding additional code in this else condition